text_extensions_for_pandas/array/test_tensor.py

```diff
@@ -224,9 +223,9 @@ def test_repr(self):
         result = repr(pd.Series(s))
         expected = textwrap.dedent(
             """\
-            0   [1 2]
-            1   [3 4]
-            2   [5 6]
+            0    [1, 2]
```

Comment:
the leading space is because that option defaults to true, we might want to look into changing that for tensors at least